src/main/java/uk/gov/digital/ho/pttg/application/HmrcClient.java

```diff
@@ -26,8 +28,10 @@
     private static final String PAYE_EMPLOYMENT = "paye";
     private static final String SA_SELF_EMPLOYMENTS = "selfEmployments";
 
-    public HmrcClient(HmrcHateoasClient hateoasClient) {
+    public HmrcClient(HmrcHateoasClient hateoasClient,
+                      @Value("#{${hmrc.paye.data.epoch:T(java.time.LocalDate).MIN}}") LocalDate payeDataEpoch) {
```

Comment:
I wonder if this defaulting should be a more explicit operation in the constructor. For one it takes away the "magic" of the Spring EL initialisation. But it will allow unit testing of the defaulting without the need for a Spring Context.